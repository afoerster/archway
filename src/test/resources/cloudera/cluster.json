{
  "name" : "odin",
  "displayName" : "Odin",
  "version" : "CDH5",
  "fullVersion" : "5.12.0",
  "maintenanceMode" : false,
  "maintenanceOwners" : [ ],
  "clusterUrl" : "http://cdh-manager:7180/cmf/clusterRedirect/cluster",
  "hostsUrl" : "http://cdh-manager:7180/cmf/clusterRedirect/cluster/hosts",
  "entityStatus" : "UNKNOWN_HEALTH",
  "uuid" : "e471ccef-1d5c-46df-a2a8-84475108c515"
}