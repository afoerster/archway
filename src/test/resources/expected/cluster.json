[{
  "id": "cluster",
  "name": "Odin",
  "status": "GOOD_HEALTH",
  "services": {
    "impala": {
      "state": "STARTED",
      "status": "GOOD",
      "name": "Impala",
      "host": "impala.example.com"
    }
  },
  "distribution": {
    "name": "CDH",
    "version": "5.11"
  }
}]